pkg/containers/client.go

```diff
@@ -62,7 +62,17 @@ func NewClient(log *logging.Logger, cgroupClient *cgroup.Client, containerdSock
 	}, nil
 }
 
-func (c *Client) Init(ctx context.Context) error {
+func (c *Client) Init(ctx context.Context, preFetchContainers bool) error {
```

Comment:
Since this is all it does this method can be removed and `FetchContainers` is called from the app.go if pre fetch is enabled.

Reply:
The whole pre-fetch feature has been dropped 👌